### PR TITLE
Ad review suggestions

### PR DESCRIPTION
**Pull request by LudwigSeitz**
Fixing even more AD review comments with the help of Ben's answers to my follow-up questions.

### CODE REVIEW

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 1]
```diff
@@ -106,7 +106,31 @@
 	"Proof-of-Possession Key Semantics for JSON Web Tokens (JWTs)" <xref target="RFC7800"/>,
 	but using CBOR <xref target="RFC7049"/> and CWTs <xref target="RFC8392"/>
 	rather than JSON <xref target="RFC8259"/> and JWTs <xref target="JWT"/>.
-     </t>
+     </t>>> Section 6
+>>
```
Comment (erdtman) [inline draft-ietf-ace-cwt-proof-of-possession.xml]: ">>" looks strange

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 2]
```diff
@@ -302,7 +326,31 @@
 	  subsequently be encrypted for use in the
 	  <spanx style="verb">Encrypted_COSE_Key</spanx> member:
 	</t>
-	<figure>
+	<figure>>> Section 6
+>>
```
Comment (erdtman) [inline draft-ietf-ace-cwt-proof-of-possession.xml]: ">>" looks strange

Comment (erdtman) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: ah, work in progress

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 1 above)
Comment (erdtman) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: ah, work in progress

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 3]
```diff
@@ -512,8 +560,10 @@
 	</t>
 	<t>
 	  Note that another means of proving possession of the key
-	  when it is a symmetric key is to encrypt the key to the recipient.
-	  The means of obtaining a key for the recipient is likewise protocol specific.
+	  exist, which could be used in conjunctions with a CWT's confirmation
```
Comment (erdtman) [inline draft-ietf-ace-cwt-proof-of-possession.xml:515]: looks good

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 4]
```diff
@@ -536,10 +586,9 @@
       </t>
       <t>
 	A recipient might not understand the <spanx style="verb">cnf</spanx> claim.
-	Applications that require
-	the proof-of-possession keys in CWTs, using the 'cnf' claim MUST ensure
-	that the parts of this specification that they use are implemented by
-	the intended recipient.
+	Applications that require the use proof-of-possession keys in CWTs,
```
Comment (erdtman) [inline draft-ietf-ace-cwt-proof-of-possession.xml:541]: looks good

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 5]
```diff
@@ -735,11 +784,13 @@
               </t>
               <t hangText='Specification Document(s):'>
                 <vspace/>
-                Reference to the document or documents that specify the parameter,
-		preferably including URIs that
-                can be used to retrieve copies of the documents.
-		An indication of the relevant
-                sections may also be included but is not required.
+                Reference to the document or documents that specify the
```
Comment (erdtman) [inline draft-ietf-ace-cwt-proof-of-possession.xml:742]: looks good

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 1 above)
Comment (selfissued) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: Indeed, this looks like a syntax error.

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 6]
```diff
@@ -106,7 +106,31 @@
...
+> verify the CWT. Do you want us to clarify this sentence?
+
+Note that I was specifically asking about "potentially the
+confidentiality"; I don't understand the intended workflow for verification
+of confidentiality (and thus which credentials are involved).  I don't
+really see how a credential held solely by the issuer could proide
+confidentiality protection when it needs to be understood by some other
+protocol participant.
+
+
```
Comment (selfissued) [inline draft-ietf-ace-cwt-proof-of-possession.xml]: It seems like you've added part of the e-mail conversation text to the draft.  Please remove it and if you want it in GitHub, please put it in an issue instead.

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 2 above)
Comment (selfissued) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: Another syntax error

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 7]
```diff
@@ -302,7 +326,31 @@
...
```
Comment (selfissued) [inline draft-ietf-ace-cwt-proof-of-possession.xml]: Please remove the e-mail conversation text from the draft source.

File: draft-ietf-ace-cwt-proof-of-possession.xml [hunk 8]
```diff
@@ -512,8 +560,10 @@
 	</t>
 	<t>
 	  Note that another means of proving possession of the key
-	  when it is a symmetric key is to encrypt the key to the recipient.
-	  The means of obtaining a key for the recipient is likewise protocol specific.
+	  exist, which could be used in conjunctions with a CWT's confirmation
+	  key. Applications making use of such alternate means are encouraged
+	  to register them in the CWT Confirmation Methods Registry
+	  <xref target="CnfReg"/>.
```
Comment (selfissued) [inline draft-ietf-ace-cwt-proof-of-possession.xml:518]: I disagree with this change, as you've deleted the text about the possibility of encrypting a symmetric key to the recipient.

**Review by selfissued — CHANGES_REQUESTED** [review]
Please revise to address my individual comments.  Especially, delete the e-mail conversation text from the document source.

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 1 above)
Comment (LudwigSeitz) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: This is really weird, my local copy of this branch does not have these syntax errors and copies of mail discussion, while if I load the remote branch, they are indeed there. I've no idea how this happened.

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 2 above)
Comment (LudwigSeitz) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: See above.

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 6 above)
Comment (LudwigSeitz) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: See above.

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 2 above)
Comment (selfissued) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: Then maybe delete this PR, delete your local repository, re-enlist, and start over from a known-good state.

Comment (LudwigSeitz) [reply draft-ietf-ace-cwt-proof-of-possession.xml]: Should be fixed now

File: draft-ietf-ace-cwt-proof-of-possession.xml (hunk 8 above)
Comment (LudwigSeitz) [reply draft-ietf-ace-cwt-proof-of-possession.xml:518]: Here is the relevant comment from the AD review:  
> While true, there are some subtleties here, namely that the recipient
> needs some way of knowing that it is the presenter (as opposed to some
> other party) that has actually performed that encryption.  In Kerberos
> we perform this key confirmation by using the encrypted (session) key
> for subsequent protocol exchanges, and if the presenter fails to process
> those exchanges then it is clear the presenter does not actually possess
> the key.  This document does not seem to describe the need for such
> subsequent confirmation (or, alternately, the consequences of failing to
> do so.)

I'm open for suggestions to better address this and not remove the part about encrypting the key to the recipient.

**Pull request merged**